CHANGELOG.md

```diff
@@ -1,5 +1,102 @@
 # Release Notes
 
+## Version 1.26.0-beta.0 - 2022-05-13
+
+Featues:
+* Add support for the QEMU driver [#13639](https://github.com/kubernetes/minikube/pull/13639)
+* Add support for building aarch64 ISO [#13762](https://github.com/kubernetes/minikube/pull/13762)
```

Comment:
Is this something we want to include, or just having supporting the QEMU driver good enough?